src/worker/requirements.txt

```diff
@@ -10,25 +10,27 @@ cachetools==5.3.1
 certifi==2023.7.22
 cffi==1.15.1
 charset-normalizer==3.2.0
+click==8.1.7
 cryptography==41.0.3
 decorator==5.1.1
+deprecation==2.1.0
 dnspython==2.4.1
 environs==9.5.0
 exceptiongroup==1.1.2
 frozenlist==1.4.0
 googleapis-common-protos==1.60.0
 greenlet==3.0.0
 grpc-gateway-protoc-gen-openapiv2==0.1.0
-grpcio
-grpcio-tools
+grpcio==1.56.0
```

Comment:
There is a weird quirk where `grpcio` and `grpcio-tools` need to have their versions undefined to avoid a dependency conflict. Please remove the version numbers from these two items.